tests/RobotFramework/tests/tedge_agent/http_server/entity_store_api.robot

```diff
@@ -126,6 +108,138 @@ Delete entity tree
     ...    {"@topic-id":"device/child2//","@parent":"device/main//","@type":"child-device"}
     ...    ${entities}
 
+Entity twin fragment apis
+    ${put}=    Execute Command
+    ...    curl -X PUT http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window -H 'Content-Type: application/json' -d '5'
+    Should Be Equal    ${put}    5
+    Should Have MQTT Messages
+    ...    te/device/main///twin/maintenance_window
+    ...    message_contains=5
+
+    ${get}=    Execute Command
+    ...    curl http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window
+    Should Be Equal    ${get}    5
+
+    ${timestamp}=    Get Unix Timestamp
+    ${http_code}=    Execute Command
+    ...    curl -o /dev/null --silent --write-out "%\{http_code\}" -X DELETE http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window
+    Should Be Equal    ${http_code}    204
+    Should Have MQTT Messages
+    ...    te/device/main///twin/maintenance_window
+    ...    date_from=${timestamp}
+    ${retained_message}=    Execute Command
+    ...    tedge mqtt sub --no-topic te/device/main///twin/maintenance_window --duration 1s
+    ...    ignore_exit_code=${True}
+    ...    strip=${True}
+    Should Be Empty    ${retained_message}
+
+Entity twin apis
+    ${new_payload}=    Set Variable    {"maintainer":"John Doe","maintenance_mode":true,"maintenance_window":5}
+    ${put}=    Execute Command
+    ...    curl -X PUT http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin -H 'Content-Type: application/json' -d '${new_payload}'
+    Should Be Equal    ${put}    ${new_payload}
+    Should Have MQTT Messages
+    ...    te/device/main///twin/maintenance_mode
+    ...    message_contains=true
+    Should Have MQTT Messages
+    ...    te/device/main///twin/maintenance_window
+    ...    message_contains=5
+
+    ${get}=    Execute Command
+    ...    curl http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin
+    Should Be Equal
+    ...    ${get}
+    ...    ${new_payload}
+
+    # Replace existing twins
+    ${timestamp}=    Get Unix Timestamp
+    ${new_payload}=    Set Variable    {"maintenance_mode":false}
+    ${put}=    Execute Command
```

Comment:
This looks to be the offending line which is causing the RF lint error (e.g. `${put}` assigned to be never used.

Reply:
Resolved by 7d725d7